package.json

```diff
@@ -51,6 +51,7 @@
     "@size-limit/preset-small-lib": "^4.9.2",
     "@typescript-eslint/eslint-plugin": "^4.15.1",
     "@typescript-eslint/parser": "^4.15.1",
+    "classnames": "^2.2.6",
```

Comment:
oh, kinda surprised that last PR worked fine without this 👀

Reply:
almost definitely a dev dependency of something else. Would only work by fluke in another project.